src/rust/engine/src/selectors.rs

```diff
@@ -59,6 +59,9 @@ impl Selector {
     )
   }
 
+  /*
+  * The product type this selector will ultimately produce.
```

Comment:
Hm. After thinking a bit, I looked at the Rust book again. It seems like line comments are preferred. And the doc comments are form of line comments. I'm changing it to line comments
https://doc.rust-lang.org/beta/book/comments.html